flang: fails to build with `fatal error: 'flang/Optimizer/Dialect/CUF/Attributes/CUFEnumAttr.h.inc' file not found`

On debian linux:

```
FAILED: tools/flang/lib/Optimizer/Dialect/CUF/CMakeFiles/obj.CUFDialect.dir/CUFDialect.cpp.o 
/opt/sccache//sccache /build/source/build-llvm/./bin/clang++ -DFLANG_INCLUDE_TESTS=1 -DFLANG_LITTLE_ENDIAN=1 -DFLANG_VENDOR="\"Debian \"" -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -Itools/flang/lib/Optimizer/Dialect/CUF -I/build/source/flang/lib/Optimizer/Dialect/CUF -I/build/source/flang/include -Itools/flang/include -Iinclude -I/build/source/llvm/include -isystem /build/source/llvm/../mlir/include -isystem tools/mlir/include -isystem tools/clang/include -isystem /build/source/llvm/../clang/include -fstack-protector-strong -Wformat -Werror=format-security -Wno-unused-command-line-argument -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -fno-semantic-interposition -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -pedantic -Wno-long-long -Wc++98-compat-extra-semi -Wimplicit-fallthrough -Wcovered-switch-default -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wsuggest-override -Wstring-conversion -Wmisleading-indentation -Wctad-maybe-unsupported -fdiagnostics-color -ffunction-sections -fdata-sections -ffile-prefix-map=/build/source/build-llvm/tools/clang/stage2-bins=../../../../ -ffile-prefix-map=/build/source/= -no-canonical-prefixes -Wno-deprecated-copy -Wno-string-conversion -Wno-ctad-maybe-unsupported -Wno-unused-command-line-argument -Wstring-conversion           -Wcovered-switch-default -Wno-nested-anon-types -O2 -DNDEBUG -g1  -fno-exceptions -funwind-tables -std=c++17 -MD -MT tools/flang/lib/Optimizer/Dialect/CUF/CMakeFiles/obj.CUFDialect.dir/CUFDialect.cpp.o -MF tools/flang/lib/Optimizer/Dialect/CUF/CMakeFiles/obj.CUFDialect.dir/CUFDialect.cpp.o.d -o tools/flang/lib/Optimizer/Dialect/CUF/CMakeFiles/obj.CUFDialect.dir/CUFDialect.cpp.o -c /build/source/flang/lib/Optimizer/Dialect/CUF/CUFDialect.cpp
In file included from /build/source/flang/lib/Optimizer/Dialect/CUF/CUFDialect.cpp:14:
In file included from /build/source/flang/include/flang/Optimizer/Dialect/CUF/CUFOps.h:12:
/build/source/flang/include/flang/Optimizer/Dialect/CUF/Attributes/CUFAttr.h:23:10: fatal error: 'flang/Optimizer/Dialect/CUF/Attributes/CUFEnumAttr.h.inc' file not found
   23 | #include "flang/Optimizer/Dialect/CUF/Attributes/CUFEnumAttr.h.inc"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

I'm still getting an error even with #e90126e in the tree:

```
In file included from /net/home/micha/projects/llvm/llvm-project/flang/lib/Optimizer/Dialect/CUF/Attributes/CUFAttr.cpp:14:
/net/home/micha/projects/llvm/llvm-project/flang/include/flang/Optimizer/Dialect/CUF/CUFDialect.h:24:10: fatal error: flang/Optimizer/Dialect/CUF/CUFDialect.h.inc: No such file or directory
   24 | #include "flang/Optimizer/Dialect/CUF/CUFDialect.h.inc"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
[162.350s  61% 1x 4514/7315] Linking CXX shared library lib/libMLIRGPUPipelines.so.19.0git
ninja: build stopped: subcommand failed.
nice ninja  645.48s user 118.48s system 470% cpu 2:42.37 total
```

--- a/flang/lib/Optimizer/Dialect/CUF/Attributes/CMakeLists.txt
+++ b/flang/lib/Optimizer/Dialect/CUF/Attributes/CMakeLists.txt
@@ -5,6 +5,7 @@ add_flang_library(CUFAttrs
   DEPENDS
   MLIRIR
   CUFAttrsIncGen
+  CUFOpsIncGen
 
   LINK_LIBS
   MLIRTargetLLVMIRExport

